A71-xds-fallback.md

```diff
@@ -117,20 +183,43 @@ file.
 
 Other approaches considered:
 
-#### Always switch to a fallback server if the primary is not available
-
-A single XdsClient is used for all targets. All cached resources are purged and
-refetched as soon as primary xDS server becomes unavailable. Major shortcoming
-of this approach is that this would put fallback server availability at risk
-due to spike in demand once the primary server goes down and all the clients
-try to refetch the resources.
-
-#### Always switch to a fallback server if the primary is not available but only if there's a need to fetch the resources that are missing from the cache.
+### Using a single XdsClient instance for all data plane targets
+
+Using a singleton XdsClient instance would cause individual channels that
+already have all of their resources cached from the primary server to switch
+to fallback resources in the following cases:
+
+- If the primary server goes down and a gRPC client then creates a new
+    xDS-enabled channel for a target for which the resources are not already
+    cached, that will trigger the client to fall back, not just for the new
+    channel but for all channels.
+- Cases where the primary server goes down while the client is in the middle
+    of processing an update (referred to as the "mid-update" case).
+    For example, let's say that the XdsClient gets an updated RouteConfig that
+    includes a new cluster that it was not previously using, thus triggering
+    the client to subscribe to the new Cluster resource, but then TD goes down
+    before sending the new Cluster resource. At this point, the XdsClient will
+    fall back to the secondary server, and it will use the fallback resources
+    instead of the cached resources that it already had from the primary server.
+
+The best compromise that we can find is to use a different XdsClient instance
+for each target, but still share those XdsClient instances between channels for
+the same target. This is not exactly the same as having each individual channel
+fall back independently, but it's close enough in practice: if there are
+multiple channels for the same target, they will all use the same decision
+about whether or not to fall back, but that should be fine, since they are also
+using the same set of xDS resources.
+
+### Using a single XdsClient but only refetch cached resources if a request was made for a non-cached resource.
```

Comment:
I don't see how this is different from the previous item.  I suggest just removing this.